import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.ResultSet;
import java.sql.SQLException;
import java.sql.Statement;

import com.mysql.cj.protocol.Resultset;
public class Self1 {

	public static void main(String[] args) throws SQLException{
		String url = "jdbc:mysql:///Company";
		String user= "root";
		String passWord ="root";
		Connection connection = DriverManager.getConnection(url,user,passWord);
		System.out.println("Connection stablished succesfully");
		Statement statement = connection.createStatement();
		System.out.println("Statement created succesfully");
		String selectQuery = "select id, name,salary,gender from Company";
		ResultSet resultset = statement.executeQuery(selectQuery);
		System.out.println("");
		System.out.println("No. of rows affected is :: "+
				rowAff);
		resultset.close();
		statement.close();

		connection.close();
		System.out.println("Resources closing.....");
	}
	
}
